src/Statistic/StatisticCollector.php

```diff
@@ -18,12 +18,15 @@ class StatisticCollector
 {
     private $client;
 
+    /**
+     * @param Client<string, Client> $client
```

Comment:
I don't understand this phpdoc, this argument is not an array

Reply:
Client is an IteratorAggregate returning an ArrayIterator of itself (see https://github.com/predis/predis/blob/main/src/Client.php#L41) thus PHPStan requires us to specify the type iterated